Sources/Datadog/RUM/RUMMonitor/Scopes/RUMSessionScope.swift

```diff
@@ -118,7 +118,17 @@ internal class RUMSessionScope: RUMScope, RUMContextProvider {
         }
 
         // Propagate command
-        viewScopes = manage(childScopes: viewScopes, byPropagatingCommand: command)
+        if !viewScopes.isEmpty {
+            viewScopes = manage(childScopes: viewScopes, byPropagatingCommand: command)
+        } else {
+            userLogger.warn(
+                """
+                A RUM event was detected, but no view is active. To track views automatically, try calling the
```

Comment:
we can also log `command` in the message, this may help the developer with debugging the problem
```swift
\(String(describing: command)) is detected but no view is active...
```

Reply:
Indeed, this can help even more. Thanks for the suggestion, I've updated the code.